SCClassLibrary/JITLib/ProxySpace/NodeMap.sc

```diff
@@ -135,7 +135,7 @@ ProxyNodeMap : NodeMap {
 			var rate = if(value.respondsTo(\rate) and:{
 				value.rate == \audio
 			}) { \audio } { \control };
-			res = res.add(ControlName(key, nil, rate, value))
+			res = res.add(ControlName(key, nil, rate, value.asUGenInput))
```

Comment:
should be `asControlInput`, because the `ControlNames` are all used outside a UGen graph.

Reply:
Thanks for catching this! Sorry for incorrectly copying your solution.